pkg/auth/authenticator/password/htpasswd/htpasswd.go

```diff
@@ -194,10 +197,11 @@ func testMD5Password(password, hash string) (bool, error) {
 }
 
 func testCryptPassword(password, hash string) (bool, error) {
-	// if len(password) > 8 {
-	// 	password = password[:8]
-	// }
-	// salt := hash[0:2]
-	// hash = hash[2:]
-	return false, errors.New("crypt password hashes are not supported")
+	testhash, err := util.Crypt(password, hash)
```

Comment:
Yes, that's correct ... and that's how it's called in lots and lots of code. Including things like pam_unix: 

http://www.openpam.org/browser/openpam/trunk/modules/pam_unix/pam_unix.c#L123